csharp/src/Drivers/BigQuery/BigQueryStatement.cs

```diff
@@ -103,8 +103,19 @@ public override QueryResult ExecuteQuery()
 
             string table = $"projects/{results.TableReference.ProjectId}/datasets/{results.TableReference.DatasetId}/tables/{results.TableReference.TableId}";
 
+            int maxStreamCount = 1;
```

Comment:
Is `1` really the best default value here? (I don't know how we ended up with 1; the documentation clearly recommends 0.

Reply:
The default value is set to 1, mainly due to two reasons:

1 The original driver setting is 1. The GBQ MSDI driver is now being used by a large number of users, and changing this setting may cause potential regression.

2 The doc does say: If unset or zero, the server will provide a value of streams so as to produce reasonable throughput. But I found in my test that this is not 100% accurate. At least in the test of reading the same table, it took 42.5 minutes when max_stream_count=1 and 46.4 minutes when max_stream_count=0. Although this may be due to other reasons such as the test table content or network fluctuations, it at least shows that potential regression on perf exists.